src/server/routes/v2/manifests.ts

```diff
@@ -25,6 +25,11 @@ export default async (fastify: FastifyInstance): Promise<void> => {
   fastify.get("/:id/:version", { schema: manifestSchema }, async (request, response) => {
     const { id, version } = request.params;
 
+    if (id == null || version == null) {
```

Comment:
this is already validated by fastify, prefer to check the input if possible using fastifys built in validation
![image](https://user-images.githubusercontent.com/18635887/91081705-6df01480-e63f-11ea-9e68-be6cc8d20d8c.png)